lib/vagrant-ovirt3/action/create_vm.rb

```diff
@@ -17,7 +17,7 @@ def call(env)
           config = env[:machine].provider_config
 
           # Gather some info about domain
-          name = (config.name.nil? ? env[:domain_name] : config.name)[0,15]
+          name = (config.name.nil? ? env[:domain_name] : config.name)[0,80]
```

Comment:
you can take this out, i'm going to make a specific commit tonight to address this issue